seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/Firefox.scala

```diff
@@ -11,6 +11,12 @@ import scala.collection.JavaConversions._
 import scala.collection.mutable.ArrayBuffer
 
 object Firefox extends SeleniumBrowser {
+
+  private final val popHijackedConsoleScript = {
+    "var callback = arguments[arguments.length - 1];" +
```

Comment:
Does this end up in the global scope? Can we avoid this with a wrapping like `(function() { ... }).call(this);`?